manifests/pg_repack.pp

```diff
@@ -36,7 +40,14 @@
   $postgresql_version = $facts['pe_postgresql_info']['installed_server_version']
   $repack_executable = "/opt/puppetlabs/server/apps/postgresql/${postgresql_version}/bin/pg_repack"
 
-  $repack_cmd = "${repack_executable} --jobs ${jobs}"
+  #Add conditionals, 
```

Comment:
The comments here before and after the `if` aren't really necessary since it's self-explanatory.

Reply:
Artifact from development, I'll remove it.